packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_filter_dialog.dart

```diff
@@ -0,0 +1,159 @@
+                  ),
+                  if (_showHelp) ...[
+                    const SizedBox(height: denseSpacing),
+                    const Text('Choose and customize the filter.\n'
+                        'List full or partial class names separated by new lines. For example:\n\n'
+                        '  package:myPackage/src/myFolder/myLibrary.dart/MyClass\n'
+                        '  MyClass\n'
+                        '  package:myPackage/src/\n\n'
+                        'Specify:\n'
+                        '  - ${ClassFilter.coreLibrariesAlias} for core classes without package prefix\n'
+                        '  - ${ClassFilter.dartAndFlutterLibrariesAlias} for "dart." and "package:" libraries published by Dart and Flutter orgs.'),
+                  ],
+                  SizedBox(height: itemSpacing),
+                  RadioButton<ClassFilterType>(
+                    label: 'Show all classes',
+                    itemValue: ClassFilterType.all,
+                    groupValue: _type,
+                    onChanged: onTypeChanged,
+                  ),
+                  SizedBox(height: itemSpacing),
+                  RadioButton<ClassFilterType>(
+                    label: 'Show all classes except:',
+                    itemValue: ClassFilterType.except,
+                    groupValue: _type,
+                    onChanged: onTypeChanged,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(left: textFieldLeftPadding),
+                    child: TextField(
+                      keyboardType: TextInputType.multiline,
+                      maxLines: null,
+                      controller: _except,
+                    ),
+                  ),
+                  SizedBox(height: itemSpacing),
+                  RadioButton<ClassFilterType>(
+                    label: 'Show only:',
+                    itemValue: ClassFilterType.only,
+                    groupValue: _type,
+                    onChanged: onTypeChanged,
+                  ),
+                  Padding(
+                    padding: EdgeInsets.only(left: textFieldLeftPadding),
+                    child: TextField(
+                      keyboardType: TextInputType.multiline,
+                      maxLines: null,
+                      controller: _only,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+      actions: [
+        DialogTextButton(
+          onPressed: () =>
+              setState(() => _loadStateFromFilter(ClassFilter.empty())),
+          child: const Text('Reset Defaults'),
+        ),
+        DialogCloseButton(
+          label: 'OK',
+          onClose: () {
+            final newFilter = ClassFilter(
+              filterType: _type,
+              except: _except.text,
+              only: _only.text,
+            );
+            if (newFilter.equals(widget.classFilter.value)) return;
```

Comment:
this is redundant so we can remove. ValueNotifier will not notify when the value is the same